### ISSUE TITLE
Request merging some features from the standalone UI to the plugin

### ISSUE THREAD

**Issue by simontuffs** [Pogadog/anomalizer-grafana-plugin#1]
One of the advantages that the Anomalizer stand-alone UI has is for exploration of metrics, allowing quick setup of filters on both the UI side and the server side. I would like to propose adding these back to the UI in the manner shown below.

<img width="1525" alt="anomalizer-mockup" src="https://user-images.githubusercontent.com/285867/187042258-b6831eda-b381-4fe8-bb86-eee0b99ce692.png">

These would be loaded from the side-bar settings when the task-bar is opened, and saved back when closed, to make persistent.  The "save" (floppy-disk) icon would save the filters to the sidebar list of filters as shown test-1, test-2 etc.

When the task bar is closed, the filters would remain accessible though the settings.

**Comment by simontuffs** [comment]
Also, it would be good to bring back the "sort-by" drop-down to this toolbar, and make the default be "features", where feature is "normalized_features" below:
```
Features:
{
    "distribution": {
        "2": "right-tailed",
        "6": "right-tailed"
    },
    "normalized_features": 0.062111174904345576
}
```

**Comment by pinky00707** [comment]
These are good ideas. I will need to think about them and how they can work with the Grafana configuration panels.

**Comment by simontuffs** [comment]
Aha, I think I'm going to retract this request for now, it is possible to get the net result by having the setting panel and visualization open at the same time. For some reason this wasn't happening when I first tested it, but I have tested in Grafana 7.0.0 and latest (8.4.5) and apart from the expected changes in how you pick a visualization, everything works nicely.

<img width="1595" alt="Screen Shot 2022-08-28 at 12 52 51 PM" src="https://user-images.githubusercontent.com/285867/187092185-99de3d18-6904-4c04-8548-b09c4d63d0fd.png">

**Comment by pinky00707** [comment]
Exactly! That's how the panel is supposed to be edited--tweak the preferences at the side and watch the panel update in realtime. What steps did you take to show the visualization on the desktop so we can add it to the troubleshooting side of the documentation?
